app/models/curator/collection.rb

```diff
@@ -6,11 +6,19 @@ class Collection < ApplicationRecord
     include Curator::Metastreams::Administratable
     include Curator::Metastreams::Workflowable
     include Curator::Mappings::Exemplary::ObjectImagable
+    include Curator::Indexable
 
     belongs_to :institution, inverse_of: :collections, class_name: 'Curator::Institution'
 
     has_many :admin_set_objects, inverse_of: :admin_set, class_name: 'Curator::DigitalObject', foreign_key: :admin_set_id, dependent: :destroy
 
     has_many :collection_members, inverse_of: :collection, class_name: 'Curator::Mappings::CollectionMember', dependent: :destroy
+
+    self.curator_indexable_mapper = Curator::CollectionIndexer.new
+
+    def exemplary_file_set
+      exemplary_image_mapping = exemplary_image_mappings.first
+      exemplary_image_mapping ? exemplary_image_mapping.exemplary_file_set : nil
+    end
```

Comment:
Move `emplary_file_set` into `Curator::Mappings::Exemplary::ObjectImagable` concern for now pending #23

Reply:
Fixed via 2075da6378d0901b1da558acbcc128391e49ea15

(I believe this also fixes #23)